sensor/common/compliance/nodescan_handler_test.go

```diff
@@ -0,0 +1,135 @@
+package compliance
+
+import (
+	"context"
+	"testing"
+
+	timestamp "github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/storage"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/goleak"
+)
+
+func assertNoGoroutineLeaks(t *testing.T) {
+	goleak.VerifyNone(t,
+		// Ignore a known leak: https://github.com/DataDog/dd-trace-go/issues/1469
+		goleak.IgnoreTopFunction("github.com/golang/glog.(*loggingT).flushDaemon"),
```

Comment:
uh oh. is this a bad leak? I see it's unresolved, but I wonder how much this affects us

Reply:
I didn't dig deeper, but there is some info on the link added. I think the leak is in a dependency of a dependency  of a dependency and it may be tricky to get rid of.